.github/workflows/zotero-bib-to-git.yml

```
@@ -12,33 +12,30 @@ jobs:
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.13'
+
+      - name: Run
+        run: uv run zotero-bib-to-gh.py
+        env:
+          ZOTERO_USER_ID: ${{ secrets.ZOTERO_USER_ID }}
+          ZOTERO_BEARER_TOKEN: ${{ secrets.ZOTERO_BEARER_TOKEN }}
+
+      - name: Commit files
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2046:warning:3:6: Quote this to prevent word splitting

(shellcheck)